### PR TITLE
Last improvements to GSoD '20 blgo post

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Preview: https://dvc-landing-gsod-v9trva5mdy8or.herokuapp.com/blog/gsod-ideas-2020

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Hi @casperdcl me again. This PR has a different link check issue. 3 URLs that work fine from browser as marked as 404 and 405. Please see https://circleci.com/gh/iterative/dvc.org/3305. Thanks

File: content/blog/2020-04-30-gsod-ideas-2020.md [hunk 1]
```diff
@@ -77,14 +77,40 @@ related to our Node.js docs web app. For example:
...
 ![](/uploads/images/2020-04-30/SciPy_India_Aman.png) _Our intern Aman took a
 several-hour long train ride to
 [talk](https://static.fossee.in/scipy2019/SciPyTalks/SciPyIndia2019%5FS011%5FStoring%5Fa%5Ffew%5Fversions%5Fof%5Fa%5F5GB%5Ffile%5Fin%5Fa%5Fdata%5Fscience%5Fproject%5F20191130.mp4)
 at [SciPy India 2019](https://scipy.in/2019)._
 
+Another star contributor who found our project via GSoD,
+[Kurian](https://github.com/kurianbenoy), closed
+[several tickets](https://github.com/iterative/dvc.org/issues?q=is%3Aissue+kurianbenoy),
+produced a DVC intro tutorial in
+[Kaggle](https://www.kaggle.com/kurianbenoy/introduction-to-data-version-control-dvc)
```
Comment (casperdcl) [inline content/blog/2020-04-30-gsod-ideas-2020.md:94]:
```
$ curl -IL  https://www.kaggle.com/kurianbenoy/introduction-to-data-version-control-dvc
HTTP/1.1 404 Not Found
...
```

looks like an upstream website issue; could add to exceptions?

File: content/blog/2020-04-30-gsod-ideas-2020.md [hunk 2]
```diff
@@ -77,14 +77,40 @@ related to our Node.js docs web app. For example:
...
+and
+[Colab](https://colab.research.google.com/drive/1O1XmUZ8Roj1dFxWTrpE55_A7lVkWfG04),
```
Comment (casperdcl) [inline content/blog/2020-04-30-gsod-ideas-2020.md:96]:
```
$ curl -IL https://colab.research.google.com/drive/1O1XmUZ8Roj1dFxWTrpE55_A7lVkWfG04
HTTP/1.1 405 Method Not Allowed
...
```

File: content/blog/2020-04-30-gsod-ideas-2020.md [hunk 3]
```diff
@@ -77,14 +77,40 @@ related to our Node.js docs web app. For example:
...
+and ended up giving a talk at
+[PyCon India](https://in.pycon.org/cfp/2019/proposals/machine-learning-model-and-dataset-versioning~dRqRb/):
```
Comment (casperdcl) [inline content/blog/2020-04-30-gsod-ideas-2020.md:98]:
```
$ curl -IL https://in.pycon.org/cfp/2019/proposals/machine-learning-model-and-dataset-versioning~dRqRb/
HTTP/1.1 405 Method Not Allowed
...
```

**Comment by shcheklein** [comment]
@casperdcl @jorgeorpinel yeah, let's merge for (don't add to exclusions since we don't even test /blog on a regular basis). I know there are a quite a few other links there that require special handling. Some of them I mitigated with user-agent, but for others we'll need to do some additional tricks (e.g. enable cookies + redirects). For those left (linkedin, reddit, kaggle, etc - sites that fight hard with scrapping) we'll need to support `website/*` (wildcard) exclusion in the list. Let's deal with this when we start testing blog content properly.

**Pull request merged**